core/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -826,7 +830,12 @@ protected void onException(Channel channel, Exception e) {
         } else if (e instanceof TcpTransport.HttpOnTransportException) {
             // in case we are able to return data, serialize the exception content and sent it back to the client
             if (isOpen(channel)) {
-                sendMessage(channel, new BytesArray(e.getMessage().getBytes(StandardCharsets.UTF_8)), () -> {}, true);
+                try {
+                    sendMessage(channel, new BytesArray(e.getMessage().getBytes(StandardCharsets.UTF_8)), () -> {
+                    }, true);
+                } catch (IOException ex) {
+                    logger.trace("failed to send error message back to client", ex);
```

Comment:
same here, trace only? can we at least have a comment explaining why it is ok to swallow these exceptions, and justifying the debug level? otherwise a reader trying to understand this code will just think "why are we swallowing exceptions"